android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -1661,59 +1090,20 @@ public void updateBuySendSwapAsset(
                     token.symbol, getResources().getDisplayMetrics().density, assetText, this, true,
                     (float) 0.5, true);
         }
-        if (buySend && (token.isErc721 || token.isNft)) {
+        if (token.isErc721 || token.isNft) {
```

Comment:
nit: can remove `token.isErc721`, `token.isNft` is enough.

Reply:
Removed now.